ultra/ultra/scenarios/common/distributions.py

```diff
@@ -295,57 +295,92 @@
                 "distribution": behavior_distribution,
                 "start_end_on_different_lanes_probability": 0.0,
                 "begin_time_init": {
-                    "func": basic_begin_time_init_func,
-                    "params": {"probability": prob_heavy},
+                    "func": burst_begin_time_init_func,
+                    "params": {
+                        "vehicle_cluster_size": (1, 2),
+                        "time_between_cluster": (60, 80),
+                        "time_for_each_cluster": 5,
+                    },
                 },
-                "has_turn": True,
-                "deadlock_optimization": True,
+                "has_turn": False,
+                "deadlock_optimization": False,
```

Comment:
Is there a specific reason for flipping these flags? I assume they introduce issues with the traffic density, but it would be good to document reasoning.

Reply:
@AlexLewandowski I agree, I did not play around with those flags. I assume that the `has_turn` flag means that there is some type of turn in the specifiied route

Reply:
Should `has_turn` be `True` here then? The route `south-east` should have a turn.